### PR TITLE
Filling in layout

### PR DESCRIPTION
**Pull request by MSedkowski**

### CODE REVIEW

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (agalkiewicz) [inline app/assets/javascripts/users.coffee]: Delete unnecessary comments.

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (agalkiewicz) [inline app/assets/stylesheets/users.scss]: as above

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
+  def new
```
Comment (agalkiewicz) [inline app/controllers/users_controller.rb]: What about 
def new; end 
form?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -2,12 +2,17 @@
 <html>
   <head>
     <title><%= full_title(yield(:title)) %></title>
-    <%= stylesheet_link_tag    'application', media: 'all',
-                                              'data-turbolinks-track' => true %>
+    <%= stylesheet_link_tag 'application', media: 'all',
```
Comment (agalkiewicz) [inline app/views/layouts/application.html.erb]: What about writing it in one line? That's less than 100 characters, so it isn't necessary to break the text in two lines.

File: config/routes.rb [hunk 5]
```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
-  root 'static_pages#home'
-  get  'static_pages/help'
-  get  'static_pages/about'
+  get 'users/new'
```
Comment (agalkiewicz) [inline config/routes.rb]: Delete this line. Look at the end of the file. You used this controller - action pair for the second time.

File: app/assets/javascripts/users.coffee (hunk 1 above)
Comment (MSedkowski) [inline app/assets/javascripts/users.coffee]: Done

File: app/assets/stylesheets/users.scss (hunk 2 above)
Comment (MSedkowski) [inline app/assets/stylesheets/users.scss]: Done

File: app/controllers/users_controller.rb (hunk 3 above)
Comment (MSedkowski) [inline app/controllers/users_controller.rb]: Corrected

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (MSedkowski) [inline app/views/layouts/application.html.erb]: Corrected

File: config/routes.rb (hunk 5 above)
Comment (MSedkowski) [inline config/routes.rb]: Corrected

**Pull request closed without merging**